compactor/compactor.go

```diff
@@ -29,8 +29,6 @@ var (
 )
 
 const (
-	checkCompactionInterval = 5 * time.Minute
```

Comment:
maybe move this to the revision compactor since it is the only one who is using it?